packages/wonder-blocks-toolbar/components/toolbar.test.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import * as React from "react";
+import {shallow} from "enzyme";
+
+import Toolbar from "./toolbar.js";
+
+describe("Toolbar", () => {
```

Comment:
These tests seem to be very low value since we the snapshot tests from the documentation and we can look at those to verify that the left/right content props got where they needed to go.  I think think we can remove these.  In the future we'll be adding visual regression tests.  For components that are purely presentation that's where we'll get value from testing.